### ISSUE TITLE
Create an archetype results repository

### ISSUE THREAD

**Issue by phylroy** [canmet-energy/btap_tasks#211]
### Description
There is a need to make our models publicly available so other may use our data. For this we would like to create a gi repository and will contain all the data generated by BTAP in a consistent format updated on a weekly basis. 

The repository will contain the results of the simulation run done on Amazon.  It will also contain a folder for powerbi projects that will enable user do datavisualize the results. 

### Approach
1. Become Familiar with the Amazon generation process. 
2. Create a private repository
2. Create a READ.md file that is formatted nicely that contained instructions on how to run the simulations on Amazon and store the results. Complete with Images of how to use PAT in the Readme.md document. 

### Waiting On
If this task is dependent on another task or NREL. Include notes Here. 

### Repositories Involved

https://github.com/canmet-energy/btap-development-environment
https://github.com/canmet-energy/docker-openstudio
https://github.com/canmet-energy/btap
https://github.com/NREL/openstudio-standards/tree/nrcan

### Select the Project Code you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	
|OEE    |P-001310.003.02.01 |TA OEE Support-1.1 Express Document Stds|Meli Stylianou
|OEE    |P-001310.003.02.02 |TA OEE Support-1.2 Product Testing|	Meli Stylianou
|OEE    |P-001310.003.02.03 |TA OEE Support-1.3 Installation Guide|Meli Stylianou
|OEE    |P-001310.003.02.04 |TA OEE Support-1.4 CCHP for heating only|	Meli Stylianou
|OEE    |P-001310.003.02.06 |TA OEE Support-1.6 In-Situ Monitoring	|Meli Stylianou
|OEE    |P-001310.003.02.07 |TA OEE Support-2.2 Roadmapping	|Meli Stylianou
|OEE    |P-001310.003.02.08 |TA OEE Support-2.3 Commercial Aspiration 	|Meli Stylianou
|OEE    |P-001310.003.02.09 |TA OEE Support-2.5 Windows	|Meli Stylianou
|OEE    |P-001787.001       |Technical Support to OEE Housing |Anil Parekh
|PERD   |P-002518.001       |BE-1 Design of HPHB |Meli Stylianou
|PERD   |P-002518.001.01	|MURBS Monitoring |Michel Tardif
|PERD   |P-002518.001.02	|Emerging Technology Assessments |Michel Tardif
|PERD   |P-002518.001.03	|Building Archetype	|Phylroy Lopez
|PERD   |P-002518.001.04	|Model Algorithms	|Jeff Blake
|PERD   |P-002518.001.05	|Design Guides |Alex Ferguson
|PERD   |P-002519.001	    |High Performance Building Envelope	|Anil Parekh
|PERD   |P-002520.001	    |New Construction Northern Housing	|Julia Purdy
|EIP	|P-002587.001	|EIP Building Envelope Retrofits	|Mark Carver
|EIP	|P-002599.001	|Field Performance of BIPV-T Technologies	|Anil Parekh
|EIP	|P-002618.001	|Technology Impact Forecasting for Canada’s Residential and Commercial Building sectors	|Alex Ferguson
|EIP	|P-002625.001	|Cold climate heat pumps for space and hot water heating	|Jeremy Sager
